ansible_events_ui/db/migrations/versions/202209021746_1285eea03d23_large_object_logs.py

```diff
@@ -0,0 +1,121 @@
+"""Add large object logs.
+
+Revision ID: 1285eea03d23
+Revises: 0a069266ef71
+Create Date: 2022-09-02 17:46:05.902097+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "1285eea03d23"
+down_revision = "93a62b2e768b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    comment = (
+        "OID of large object containing log(s). "
+        "This value will be created by a trigger if not supplied."
+    )
+    op.add_column(
+        "activation_instance",
+        sa.Column(
+            "log_id",
+            postgresql.OID(),
+            nullable=True,
+            comment=comment,
+        ),
+    )
+
+    # Create trigger function to create log_id if that column is null
+    op.execute(
+        """
```

Comment:
Raw SQL should be wrapped in `sa.text()`

Reply:
Again, this is optional. I did not find any reference requiring this.

Reply:
Please see my comment above. This is not optional.